src/app/shared/select/select-item/select-item.component.html

```diff
@@ -27,6 +27,7 @@
       <span class="ml-sm-1 text-nowrap">
         {{selectedItem.value}}
       </span>
+      <i *ngIf="removable && !disabled" class="fa-fw fal fa-times mx-1 clear" (click)="$event.preventDefault(); $event.stopPropagation(); onRemove()"></i>
```

Comment:
The code is more readable when we do not call multiple functions directly in the template.